libs/dh/wholesale/feature-calculations/src/lib/calculations.component.scss

```diff
@@ -17,8 +17,21 @@
 
 @use "@energinet-datahub/watt/utils" as watt;
 
+:host {
+  position: absolute;
+  inset: 0;
+}
+
 .wrapper {
   @include watt.space-inset-l;
+  height: 100%;
+}
+
+watt-card {
+  display: grid;
+  grid-template-rows: auto auto minmax(0, 1fr);
+  grid-template-columns: 100%;
+  height: 100%;
 }
```

Comment:
question: does this "pattern" need to be used throughout all pages in order to achieve the same/similar layout? So `position: absolute` on the outermost page element, `height: 100%` on a wrapper, and `display: grid` on a child? Not relevant currently but we should consider how to make sure this is applied correctly throughout the app.

Reply:
Yep, pretty much. The `position: absolute` could also be replaced by `height: 100%` if desired. Perhaps we should consider creating a dh ui library for handling this specific table layout? 🙂

Reply:
A library, or predefined CSS classes. Let's keep an eye in which direction things are going, then decide on a plan. But it'd be nice to apply the same style throughout.